sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -541,6 +541,24 @@ int sys_setsockopt(int fd, int layer, int number, const void *buffer, socklen_t
 	return 0;
 }
 
+int sys_sockname(int fd, struct sockaddr *addr_ptr, socklen_t max_addr_length,
+		socklen_t *actual_length) {
+	auto ret = do_syscall(NR_getsockname, fd, addr_ptr, max_addr_length);
```

Comment:
Are we using `SYS_` or `NR_`?

Reply:
This is an artifact from before the syscall PR. It gets changed in a later commit touching `sys_sockname`. In order to avoid excessively force-pushing to this PR, I'd leave this commit as is.

Reply:
Can't you just push another fixup?

Reply:
The problem is that a later commit rectifies the issue, which would necessitate a rebase.

Reply:
I can, however, note this and fix it during the rebase that I'll be doing for squashing the fixups.